api/core/workflow/utils/condition/entities.py

```diff
@@ -39,7 +39,7 @@ class SubCondition(BaseModel):
 
 class SubVariableCondition(BaseModel):
     logical_operator: Literal["and", "or"]
-    conditions: list[SubCondition] = Field(default=list)
+    conditions: list[SubCondition] = Field(default=[])
```

Comment:
Can we also use default_factory here?

Reply:
Sure. Changed to use `default_factory=list` here.